pkg/handler/asset.go

```diff
@@ -244,6 +244,9 @@ func (u *UserSelectHandler) proxyAsset(asset model.PermAsset) {
 			}
 			tokenInfo = reviewHandler.tokenInfo
 		default:
+			msg := lang.T("Unknown error code: %s, detail: %s")
+			utils.IgnoreErrWriteString(u.h.term, fmt.Sprintf(msg, tokenInfo.Code, tokenInfo.Detail))
+			utils.IgnoreErrWriteString(u.h.term, utils.CharNewLine)
 			logger.Errorf("Create connect token and auth info failed: %s %s", tokenInfo.Code, tokenInfo.Detail)
 			return
 		}
```

Comment:
The code does not contain any known issues or improvements. However, it could be more streamlined by using a single return statement after checking the case for all types of assets. Also, consider if adding a comment at `default case` provides additional help to someone reading this function, such that they can understand what will happen when no specific case matches. Here is an example:

```go
func (u *UserSelectHandler) proxyAsset(asset model.PermAsset) {
	if aclCheckType(u, &asset) && !hasTokenError(&tokenInfo) {

```

This change should improve readability and maintainability without altering functionality.

Let me know if you have another question!